mpas_analysis/shared/io/mpas_reader.py

```diff
@@ -195,7 +195,8 @@ def _parse_dataset_time(ds, inTimeVariableName, calendar,
         timeStrings = [''.join(xtime.astype('U')).strip()
                        for xtime in timeVar.values]
         for i, timeString in enumerate(timeStrings):
-             if timeString[8:] == '15_00:00:00' and i +1 < len(timeStrings):
+             if (timeString == '0000-01-15_00:00:00' and
+                     i + 1 < len(timeStrings)):
```

Comment:
Hmm, parentheses make this more complicated...
```suggestion
             if timeString == '0000-01-15_00:00:00' and \
                     i + 1 < len(timeStrings):
```